scss/bitstyles/layout/grid/grid.stories.mdx

```diff
@@ -2,143 +2,183 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
 
-### Dependencies
+<Canvas>
+  <Story name="l-grid (list)">
+    {`
+      <ul class="l-grid o-list-reset u-padding-m u-bg--gray-10">
+        <li class="u-bg--background u-padding-m o-card">Grid child 1</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 2</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 3</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 4</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+</Canvas>
 
-- `layout/width`
+## Columns
 
-## Grid system — base
+Most likely you’ll want to also add a grid-column class, to layout the grid children next to each other.
 
-In the base grid system, columns are of proportional widths (a percentage of the full width) but not responsive — that percentage doesn’t change when the viewport does.
+<Canvas isColumn>
+  <Story name="l-grid--2-col">
+    {`
+      <ul class="l-grid l-grid--2-col u-padding-m u-bg--gray-10 o-list-reset">
+        <li class="u-bg--background u-padding-m o-card">Grid child 1</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 2</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 3</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 4</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+  <Story name="l-grid--3-col">
+    {`
+      <ul class="l-grid l-grid--3-col u-padding-m u-bg--gray-10 o-list-reset">
+        <li class="u-bg--background u-padding-m o-card">Grid child 1</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 2</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 3</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 4</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m o-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+</Canvas>
 
-This means they cannot change from being 3-column on larger screens, to one  stacked column on small screens (if you can, try resizing your browser window to see that the number of columns in the example below doesn’t change). See the Responsive Grid below for information on how to make your grid responsive.
+### Responsive columns
 
-<Canvas>
-  <Story name="Grid">
+Column classes can be applied at specific breakpoints using the `@breakpoint` suffix on the class e.g. `l-grid--3-col@m`. This can be used to acheive a single column layout on small screens, which will break into columns when there’s enough space.
```

Comment:
```suggestion
Column classes can be applied at specific breakpoints using the `@breakpoint` suffix on the class e.g. `l-grid--3-col@m`. This can be used to achieve a single column layout on small screens, which will break into columns when there’s enough space.
```